app-backend/app/src/main/scala/token/Token.scala

```diff
@@ -27,8 +30,44 @@ object TokenService extends Config {
     Authorization(GenericHttpCredentials("Bearer", auth0Bearer))
   )
 
+  val authBearerTokenCache: AsyncLoadingCache[Int, ManagementBearerToken] =
+    Scaffeine()
+      .expireAfterWrite(12.hour)
```

Comment:
Clever idea. My instincts are to take this down to an hour, but not really sure I can come up with a scenario where 12 would lead to a problem.